php/fm-overlays-post-type.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * fm-overlays-post-type.php
+ *
+ * @created     1/6/16 3:42 PM
+ * @author      Alley Interactive
+ * @package     fm-overlays
+ * @description Set up the custom post type for the overlays
+ *
+ */
+if ( ! class_exists( 'Fm_Overlays_Post_Type' ) ) :
```

Comment:
I'd skip the `class_exists()` check on these two classes. If there was a conflict, it would be good to let the app crash so the developer knows about it.